### PR TITLE
pagination category

### PR DESCRIPTION
**Pull request by arf-hamza**

### CODE REVIEW

File: tp-3/src/app/[locale]/categories/page.tsx [hunk 1]
```diff
@@ -2,20 +2,28 @@
...
   getApiCategory,
   postApiCategory,
   putApiCategory,
   deleteApiCategory,
 } from "@/api/category.api";
-import {
-  Box,
-} from "@mui/material";
+
+import { Box, Button, Grid } from "@mui/material";
```
Comment (glitterbabi) [inline tp-3/src/app/[locale]/categories/page.tsx:11]: it's cleaner this way, good idea to put it on the same line hehe : )

File: tp-3/src/app/[locale]/categories/page.tsx [hunk 2]
```diff
@@ -41,30 +49,105 @@ const CategoryPage = () => {
...
+    setCurrentPage(page);
+  };
+
+  const indexOfLastCategories = currentPage * categoriesPerPage;
+  const indexOfFirstCategries = indexOfLastCategories - categoriesPerPage;
+  const currentCategories = categories.slice(
+    indexOfFirstCategries,
+    indexOfLastCategories
+  );
+
```
Comment (glitterbabi) [inline tp-3/src/app/[locale]/categories/page.tsx:93]: your code is simple to follow nice job

File: tp-3/src/app/[locale]/products/page.tsx [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
 import React, { useEffect, useState } from "react";
-import { Button, Box, Grid } from "@mui/material";
+import "@/app/globals.css";
+import {
+  Typography,
+  Button,
+  Box,
+  Grid,
+} from "@mui/material";
+
```
Comment (glitterbabi) [inline tp-3/src/app/[locale]/products/page.tsx:10]: maybe we could put all of those on the same line like you did above (or maybe not it's a nitpick it doesn't really matter hahah)

File: tp-3/src/app/[locale]/products/page.tsx [hunk 4]
```diff
@@ -1,13 +1,19 @@
...
 import {
   getApiProducts,
   postApiProduct,
   putApiProduct,
   deleteApiProduct,
 } from "@/api/product.api";
-import MyMenu from "@/components/molecules/my-menu/my-menu";
 import MyCardProduct from "@/components/molecules/card-product/my-card";
```
Comment (glitterbabi) [inline tp-3/src/app/[locale]/products/page.tsx:17]: is the menu still going to show up on this page or not? if not we can fix it in another update hehe

**Review by glitterbabi — APPROVED** [review]

File: tp-3/src/app/[locale]/products/page.tsx (hunk 4 above)
Comment (arf-hamza) [reply tp-3/src/app/[locale]/products/page.tsx:17]: yes you have right 👍

File: tp-3/src/app/[locale]/products/page.tsx (hunk 3 above)
Comment (arf-hamza) [reply tp-3/src/app/[locale]/products/page.tsx:10]: this was the suggestions from prettier when i do do put documentation in form the code changes like that haha

**Pull request merged**